NextcloudTalk/ChatViewController.swift

```diff
@@ -81,6 +81,28 @@ import UIKit
         return button
     }
 
+    func startCall(withVideo video: Bool, silently: Bool, button: BarButtonItemWithActivity) {
+        if self.room.recordingConsent {
+            let alert = UIAlertController(title: "⚠️" + NSLocalizedString("The call might be recorded", comment: ""),
+                                          message: NSLocalizedString("The recording might include your voice, video from camera, and screen share. Your consent is required before joining the call.", comment: ""),
+                                          preferredStyle: .alert)
+
+            alert.addAction(UIAlertAction(title: NSLocalizedString("Give consent and join call", comment: "Give consent to the recording of the call and join that call"),
+                                          style: .default,
+                                          handler: {(_: UIAlertAction!) in
+                CallKitManager.sharedInstance().startCall(self.room.token, withVideoEnabled: video, andDisplayName: self.room.displayName, silently: silently, recordingConsent: true, withAccountId: self.room.accountId)
+            }))
+
+            alert.addAction(UIAlertAction(title: NSLocalizedString("Cancel", comment: ""), style: .cancel, handler: {(_: UIAlertAction!) in button.hideIndicator()
+            }))
```

Comment:
```suggestion
            alert.addAction(.init(title: NSLocalizedString("Give consent and join call", comment: "Give consent to the recording of the call and join that call"), style: .default) { _ in
                CallKitManager.sharedInstance().startCall(self.room.token, withVideoEnabled: video, andDisplayName: self.room.displayName, silently: silently, recordingConsent: true, withAccountId: self.room.accountId)
            })

            alert.addAction(.init(title: NSLocalizedString("Cancel", comment: ""), style: .cancel) { _ in
                button.hideIndicator()
            })
```

Should be able to shorten it like this?